mxfusion/components/distributions/gamma.py

```diff
@@ -0,0 +1,134 @@
+import numpy as np
+import mxnet as mx
+from ...common.config import get_default_MXNet_mode
+from ..variables import Variable
+from .univariate import UnivariateDistribution, UnivariateLogPDFDecorator, UnivariateDrawSamplesDecorator
+from .distribution import Distribution, LogPDFDecorator, DrawSamplesDecorator
+from ..variables import is_sampled_array, get_num_samples
+from ...util.customop import broadcast_to_w_samples
+
+
+class Gamma(UnivariateDistribution):
+    """
+    Gamma distribution.
+    Takes dependency on Scipy to compute the log-gamma function.
+
+    :param a: the alpha parameter of the Gamma distribution.
+    :type a: Variable
+    :param a: beta parameter of the Gamma distribution.
+    :type b: Variable
+    :param rand_gen: the random generator (default: MXNetRandomGenerator).
+    :type rand_gen: RandomGenerator
+    :param dtype: the data type for float point numbers.
+    :type dtype: numpy.float32 or numpy.float64
+    :param ctx: the mxnet context (default: None/current context).
+    :type ctx: None or mxnet.cpu or mxnet.gpu
+    """
+    def __init__(self, a, b, mean_variance_parameterized=False, rand_gen=None, dtype=None, ctx=None):
```

Comment:
This is confusing. Maybe it is better to create another distribution called GammaMeanVariance and inherit from Gamma distribution.